.github/workflows/pr-title-check.yml

```diff
@@ -0,0 +1,41 @@
+name: "Lint PR"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
```

Comment:
One of the challenges of using the `edited` and `synchronize` types are that they can happen in quick succession (e.g., editing the PR description multiple times), leading to the workflow being triggered many times. One option to address this is to limit it to one concurrent run per PR by adding this:

```
# PR updates can happen in quick succession leading to this
# workflow being trigger a number of times. This limits it
# to one run per PR.
concurrency:
  group: ${{ github.workflow }}-${{ github.ref }}
```

Reply:
added!

Reply:
Would this cause edits to be missed aka I edit the PR title but want to change it again but change it the second time to something not valid, how dose it behave in this case for the check?

Reply:
My understanding is that it will [cancel older runs in favour of the most recent run](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-only-cancel-in-progress-jobs-or-runs-for-the-current-workflow). So in that scenario, the newest workflow would check the current state of the PR, and fail for the invalid title.

Reply:
Yup makes sense